csrc/quantization/cutlass_w8a8/scaled_mm_c3x.cu

```diff
@@ -116,6 +116,43 @@ struct ScaledEpilogue {
   }
 };
 
+template <typename ElementAcc, typename ElementD, typename EpilogueDescriptor>
+struct ScaledEpilogueAzp
+    : private ScaledEpilogue<ElementAcc, ElementD, EpilogueDescriptor> {
+ private:
+  using SUPER = ScaledEpilogue<ElementAcc, ElementD, EpilogueDescriptor>;
+  using ScaleA = typename SUPER::ScaleA;
+  using ScaleB = typename SUPER::ScaleB;
+  using EVTCompute0 = typename SUPER::EVTCompute0;
+
+  using Compute1 = cutlass::epilogue::fusion::Sm90Compute<
+      cutlass::multiply_add, ElementD, float,
+      cutlass::FloatRoundStyle::round_to_nearest>;
+
+  using Bias = cutlass::epilogue::fusion::Sm90ColOrScalarBroadcast<
+      0 /*Stages*/, typename EpilogueDescriptor::TileShape, float,
+      Stride<Int<1>, Int<0>, Int<0>>>;
```

Comment:
Ditto, this should be `Sm90ColBroadcast`. Also for `Sm90ColBroadcast`, there's a template parameter to turn off the code that does a scalar broadcast  and I think we should use that